AUTHORS

```diff
@@ -464,4 +464,9 @@ Tanu Hari Dixit <tokencolour@gmail.com>
 Shekhar Prasad Rajak <shekharrajak@live.com>
 Aqnouch Mohammed <aqnouch.mohammed@gmail.com>
 Arafat Dad Khan <arafat.da.khan@gmail.com>
+Boris Atamanovskiy <shaomoron@gmail.com>
+Arafatk <arafat@iitkpg.ac.in>
 Sam Tygier <sam.tygier@hep.manchester.ac.uk>
+alkis akritas <akritas@uth.gr>
+Jai Luthra <me@jailuthra.in>
+MSeeker <Seeker1995@gmail.com>
```

Comment:
Also update .mailmap so that these are full names (with proper capitalization) where possible.